test_conformance/events/test_event_dependencies.cpp

```diff
@@ -275,6 +276,10 @@ int test_event_enqueue_wait_for_events_run_test(
 
     for (loop_count = 0; loop_count < TEST_COUNT; loop_count++)
     {
+        clEventWrapper first_dependency = (loop_count == 0) ? pre_loop_event : last_loop_event;
+        clEventWrapper event[TEST_COUNT*4+2]; // If we usemarkers we get 2 more events per iteration
```

Comment:
Thanks for making these changes, this is looking like a nice improvement.

I think we can do better still, and make this array of size 4 (or however many events we actually need in the loop) rather than `[TEST_COUNT*4+2]`  and move the `event_count` variable into the loop to increment through those 4 events.

Reply:
> Thanks for making these changes, this is looking like a nice improvement.
> 
> I think we can do better still, and make this array of size 4 (or however many events we actually need in the loop) rather than `[TEST_COUNT*4+2]` and move the `event_count` variable into the loop to increment through those 4 events.

thank you !!